exporter/signalfxexporter/internal/utils/histogram_utils_test.go

```diff
@@ -302,9 +324,14 @@ func TestHistogramsAreRetrieved(t *testing.T) {
 		t.Run(tt.name, func(t *testing.T) {
 			md := tt.inMetricsFunc()
 			gotMetrics, gotCount := GetHistograms(md)
-
 			assert.Equal(t, tt.wantMetricCount, gotCount)
-			assert.Equal(t, tt.wantMetrics(), gotMetrics)
+			if tt.wantMetricCount == 0 {
+				assert.Equal(t, tt.wantMetrics(), gotMetrics)
+			} else {
```

Comment:
I'd say this part is redundant. `pmetrictest.CompareMetrics` should do the job even if wantMetricCount == 0, right?

Reply:
with no histograms, we return uninitialized pmetric.Metrics{} and CompareMetrics does not check for nil